testsuite/features/init_clients/allcli_update_activationkeys.feature

```diff
@@ -48,8 +48,8 @@ Feature: Update activation keys
   Scenario: Update SLE key with synced base product
     When I follow the left menu "Systems > Activation Keys"
     And I follow "SUSE Test Key x86_64" in the content area
-    And I wait until I do not see "Loading..." text
     And I select "SLE-Product-SLES15-SP4-Pool for x86_64" from "selectedBaseChannel"
+    And I wait until I do not see "Loading..." text
     And I include the recommended child channels
```

Comment:
I understand adding it as a new step, but is it safe to just move it? won't the previous steps possibly fail for the same reason again?

Reply:
I was about to ask same question :stuck_out_tongue: